manager/README.md

```diff
@@ -48,6 +48,7 @@ Paths below prefixed with `/api/transporter/v1/transfers/{request-id}`.
 
 | Verb | Path | Description |
 | ---- | ---- | ----------- |
+| PUT | `/reconsider/{transfer-id}` | Reset the state of a specific failed transfer in a request to 'Pending'. |
```

Comment:
Nit: I think `/detail/{transfer-id}/reconsider` would be more REST-idiomatic (apart from the terrible `detail` name; I opened DSPDC-490 for next sprint to clean these routes up). Ok to leave it for now if you want.